packages/kbn-pr-utils/src/cli.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  RunWithCommands,
+  RunContext,
+  Command as GenericCommand,
+  createFailError,
+} from '@kbn/dev-utils';
+import { PrApi } from './pr_api';
+import { ForceEsDocFailuresCommand } from './commands/force_es_doc_failures';
+
+/**
+ * Define additional properties attached to the runContext which will be provided to all commands
+ */
+function extendContext({ flags, log }: RunContext) {
+  const token = flags['github-token'];
+  if (!token || typeof token !== 'string') {
+    throw createFailError('--github-token is required');
```

Comment:
Thoughts on supporting `GH_TOKEN`, `GITHUB_TOKEN`?

Reply:
Not opposed, just generally find that I prefer to specify it per execution. I'll add it though because why not?!